Cargo.toml

```diff
@@ -9,6 +9,9 @@ edition = "2021"
 
 [profile.release]
 lto = "thin"
+opt-level = 3
+codegen-units = 1
+strip = true
```

Comment:
reduces the binary size from `~120mb` to `~70mb`. Compile times are a bit longer though